pkgs/applications/misc/gallery-dl/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "gallery_dl";
+  version = "1.8.2";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "6c5995b7d24dfaae7bdf71b9261e5044b01adbd5d5302aaff9ac4a30bbceedb6";
+  };
+
+  doCheck = false;
+  propagatedBuildInputs = with python3Packages; [ requests ];
+
+  meta = {
+    description = "Command-line program to download image-galleries and -collections from several image hosting sites";
+    homepage = https://github.com/mikf/gallery-dl;
```

Comment:
We probably will remove the unquoted url syntax in the near future: https://github.com/NixOS/rfcs/pull/45
And sorry for the delay, I didn't get a notification when you force pushed. Can you ping me with @lassulus after the next push and I will merge this :)

Reply:
Ah well, I guess it's not finalized yet and we can change it later. gonna merge it now